Makefile

```diff
@@ -1,10 +1,38 @@
 GOPATH ?= ${shell go env GOPATH}
 
 
+# This is how we want to name the binary output
+BINARY=cloud-gate
+#
+# # These are the values we want to pass for Version and BuildTime
+VERSION=0.5.0
+
 all:
 	@cd $(GOPATH)/src; go install github.com/Symantec/cloud-gate/cmd/*
 
 
+get-deps:
+	go get -t ./...
+
+clean:
+	rm -f bin/*
+	rm -f cloud-gate-*.tar.gz
+
+${BINARY}-${VERSION}.tar.gz:
+	mkdir ${BINARY}-${VERSION}
+	rsync -av --exclude="config.yml" --exclude="*.pem" --exclude="*.out" lib/ ${BINARY}-${VERSION}/lib/
+	rsync -av --exclude="config.yml" --exclude="*.pem" --exclude="*.out" --exclude="*.key" cmd/ ${BINARY}-${VERSION}/cmd/
+	rsync -av  misc/ ${BINARY}-${VERSION}/misc/
+	cp LICENSE Makefile cloud-gate.spec README.md ${BINARY}-${VERSION}/
```

Comment:
Yes, I understand why the binaries and specfile are copied in, and perhaps `Makefile`.
Why is `LICENSE` and `README.md` being copied?

Reply:
@rgooch Because `LICENSE` must be included with the distribution. `README.md` is not required, but may include useful information, especially if that's the closest thing to documentation that exists.